CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
+  * AKS version [`1.31`](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/) is now generally available. Please check the release tracker for when your region will receive the GA update. Some regions may not receive this update until later in November.
+  * The first official patch version of LTS 1.27 is being released.
+ 
+* Preview features:
+  * We've added a new way to optimize your upgrade process drain behavior. By default, a node drain failure causes the upgrade operation to fail, leaving the undrained nodes in a schedulable state, this behavior is called `Schedule`. Alternatively, you can select the `Cordon` behavior, which skips nodes that fail to drain by placing them in a quarantined state, labeling them `kubernetes.azure.com/upgrade-status:Quarantined` and proceeds with upgrading the remaining nodes. This ensures that all nodes are either upgraded or quarantined. This approach allows you to troubleshoot drain failures and gracefully manage the quarantined nodes.
+  * You can now [block pod access to the Azure Instance Metadata Service (IMDS) endpoint](https://learn.microsoft.com/azure/aks/imds-restriction) to enhance security.
+    
+* Bug fixes:
+ 
+* Behavior change:
+  * You can now fine-tune supported models on KAITO version [0.3.1](https://github.com/Azure/kaito/releases/tag/v0.3.1) with the AI toolchain operator add-on on your AKS cluster.
+  * Beta APIs will be disabled for newly created clusters after November 30, 2024.
+  * Virtual Machine nodepools will not be able to use custom VNets when if the cluster is using system-assigned identity.
+  * LTS no longer do defaulting on autoUpgrader profile.
+  * This release adds the below plugin blocks to coredns corefile by default.
+    * `template ANY ANY reddog.microsoft.com {      rcode NXDOMAIN    }`
+    * `template ANY ANY internal.cloudapp.net {      match "^(?:+.){4,}internal.cloudapp.net.$"      rcode NXDOMAIN      fallthrough    }`
+    * The first plugin block is designed to return NXDOMAIN for queries to reddog.microsoft.com domain. This is useful to prevent coredns from sending queries to AzureDNS for reddog.microsoft.com domain.
+    * The second plugin block is designed to return NXDOMAIN for queries to internal.cloudapp.net domain, except for domains with the required number of components following this pattern: ..*.internal.cloudapp.net. Example - Allowed: aks-agentpool-xxxx-vmssxxxxxx.xxxxxx.xx.internal.cloudapp.net . Example - Not allowed: test.aks-agentpool-xxxx-vmssxxxxxx.xxxxxx.xx.internal.cloudapp.net
+  
+* Component updates:
+  * The KEDA addon will be bumped to version 1.15 on AKS 1.32 instead of AKS 1.31.
+  * The CoreDNS addon will use v1.9.4-2 for all existing clusters and clusters created for versions up until 1.30. For kubernetes version 1.31+, CoreDNS will use v1.11.3-2
```

Comment:
nit: can we change 'will use' to 'updated to' for consistency with format of component updates points from previous releases?